docs/sources/project/get-help.md

```diff
@@ -2,103 +2,127 @@ page_title: Where to chat or get help
+        IRC a direct line to our most knowledgeable Docker users.
+        The <code>#docker</code> and <code>#docker-dev</code> group on 
+        <strong>irc.freenode.net</strong>. IRC was first created in 1988. 
+        So, it is a rich chat protocol but it can overwhelm new users. You can search
+        <a href="https://botbot.me/freenode/docker/#" target="_blank">our chat archives</a>.
+      </p>
+      Read our <a href="#irc-quickstart">IRC quickstart guide</a> below for 
+        an easy way to get started.
+    </td>
   </tr>
   <tr>
-    <td class="tg-031e">Google Groups</td>
-    <td class="tg-031e">There are two groups. <a href="https://groups.google.com/forum/#!forum/docker-user" target="_blank">Docker-user</a> is for people using Docker containers. The <a href="https://groups.google.com/forum/#!forum/docker-dev" target="_blank">docker-dev</a> group is for contributors and other people contributing to the Docker project.</td>
+    <td>Google Groups</td>
+    <td>
+      There are two groups.
+      <a href="https://groups.google.com/forum/#!forum/docker-user" target="_blank">Docker-user</a>
+      is for people using Docker containers. 
+      The <a href="https://groups.google.com/forum/#!forum/docker-dev" target="_blank">docker-dev</a> 
+      group is for contributors and other people contributing to the Docker 
+      project.
+    </td>
   </tr>
   <tr>
-    <td class="tg-031e">Twitter</td>
-    <td class="tg-031e">You can follow <a href="https://twitter.com/docker/" target="_blank">Docker's twitter</a> to get updates on our products. You can also tweet us questions or just share blogs or stories.</td>
+    <td>Twitter</td>
+    <td>
+      You can follow <a href="https://twitter.com/docker/" target="_blank">Docker's twitter</a>
+      to get updates on our products. You can also tweet us questions or just 
+      share blogs or stories.
+    </td>
   </tr>
   <tr>
-    <td class="tg-031e">Stackoverflow</td>
-    <td class="tg-031e">Stackoverflow has over 7000K Docker questions listed. We regularly monitor <a href="http://stackoverflow.com/search?tab=newest&q=docker" target="_blank">Docker questions</a> and so do many other knowledgeable Docker users.</td>
+    <td>Stack Overflow</td>
+    <td>
+      Stack Overflow has over 7000K Docker questions listed. We regularly 
+      monitor <a href="http://stackoverflow.com/search?tab=newest&q=docker" target="_blank">Docker questions</a>
+      and so do many other knowledgeable Docker users.
+    </td>
   </tr>
 </table>
 
 
 ## IRC Quickstart
 
-IRC can also be overwhelming for new users. This quickstart shows you the easiest way to connect to IRC. 
+IRC can also be overwhelming for new users. This quickstart shows you 
+the easiest way to connect to IRC. 
 
 1. In your browser open <a href="http://webchat.freenode.net" target="_blank">http://webchat.freenode.net</a>
 
-	![Login screen](/project/images/irc_connect.png)
+    ![Login screen](/project/images/irc_connect.png)
```

Comment:
I think that's a GitHub weirdness. I just ran/started make docs; you can see it here http://docs.dockr.nl:8000/project/get-help/ (might be offline later :D)